desktop-exporter/app/routes/trace-view.tsx

```diff
@@ -96,14 +97,28 @@ export default function TraceView() {
   );
 
   return (
-    <div className="traceview">
-      <Header traceID={traceData.traceID} />
-      <WaterfallView
-        spans={traceData.spans}
-        selectedSpanID={selectedSpanID}
-        setSelectedSpanID={setSelectedSpanID}
-      />
-      <DetailView span={selectedSpan} />
-    </div>
+    <Grid
+      templateAreas={`"header header"
+                       "main detail"`}
+      gridTemplateColumns={"1fr 250px"}
```

Comment:
Just as a note the CSS version was `3fr 250px`, though I am not sure if this difference matters

Reply:
Yeah, I did it wrong last time. Since detail is fixed width, I just need the rest to be 1fr. :)